sdk/aqueduct/aqueduct_client.py

```diff
@@ -33,6 +34,21 @@
 import __main__ as main
 import os
 
+def apikey() -> str:
+    """
+    Get the local host API key if the server is running locally. 
+
+    Returns:
+        The local API key.
+    """
+    server_directory = os.path.join(os.environ["HOME"], ".aqueduct", "server")
+    config_file = os.path.join(server_directory, "config", "config.yml")
+    with open(config_file, "r") as f:
+        try:
+            return str(yaml.safe_load(f)['apiKey'])
+        except yaml.YAMLError as exc:
+            print(exc)
```

Comment:
Instead of just printing out the exception trace, let's in addition print out a warning saying "This API works only when you are running the server and the SDK on the same machine.", because if the user runs into this issue, it's likely that the server is a remote one, so we can't find the api key locally.